presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -196,11 +195,6 @@ public SystemSessionProperties(
                         "Use resources which are not guaranteed to be available to the query",
                         false,
                         false),
-                stringSessionProperty(
```

Comment:
Changes in this file should probably me moved to "Remove processing_optimization" commit